docs/src/usage.md

```diff
@@ -310,11 +310,11 @@ The parameters that sources need differ per source type. Most sources need a URL
 :class: only-dark
 ```
 
-Source parameter (URL's, usernames, passwords, etc.) changes can be applied to different scopes: to just the source being edited or to multiple sources that have the same type and value as the one being edited. When applying the change to multiple sources, the user can change all sources that have the same type and value of a metric, of a subject, of a report, or of all reports.
+Source parameter (URLs, usernames, passwords, etc.) changes can be applied to different scopes: to just the source being edited or to multiple sources that have the same type and value as the one being edited. When applying the change to multiple sources, the user can change all sources that have the same type and value of a metric, of a subject, of a report, or of all reports.
 
 #### Deleting sources
 
-To delete a metric's source, expand the metric in the metric table, select the "Sources" tab and click the "Delete source" button. The source is deleted and no longer used to measure the metric.
+To delete a source, expand the metric in the metric table, select the "Sources" tab and click the "Delete source" button. The source is deleted and no longer used to measure the metric.
```

Comment:
The usage of sequencer connectors in English is a bit weird. I think it should be something like `In order to delete a source, first expand the metric in the metric table. Then, select the "Sources" tab and click the "Delete source button".`